ddtrace/monkey.py

```diff
@@ -263,12 +263,16 @@ def _attempt_patch_module(module):
             imported_module = importlib.import_module(path)
         except ImportError:
             # if the import fails, the integration is not available
-            raise PatchException("integration '%s' not available" % path)
+            raise ModuleNotFoundException(
+                "integration '%s' not available in ddtrace.contrib, module will not have traces available" % path
```

Comment:
I wonder if we can be more explicit like `integration module %s does not exist, module will not have tracing available`

since `path` is `ddtrace.contrib.<module>` could be good enough?

🤔 would like others thoughts too